cache/lru.go

```diff
@@ -134,3 +134,8 @@ func (l *LRUCache) Fetch(ctx context.Context, keys []string, opts ...Option) (re
 func (l *LRUCache) Name() string {
 	return "in-memory-" + l.name
 }
+
+func (l *LRUCache) Delete(_ context.Context, _ string) error {
+	// no-op
+	return nil
```

Comment:
We should implement it here.

Reply:
Added

Reply:
I think this also needs to call delete on the underlying cache `l.c.Delete`, right?

Reply:
That's true :palm-face:. Added.